### PR TITLE
Rename `text_layout` example to `flex_layout`

### PR DESCRIPTION
**Pull request by doup**
# Objective

- Rename `text_layout` example to `flex_layout` to better reflect the example purpose
- `AlignItems`/`JustifyContent` is not related to text layout, it's about child nodes positioning

## Solution

- Rename the example

### CODE REVIEW

**Comment by ickshonpe** [comment]
I think I originally named it `flex_layout` even. I changed it because I thought most users probably don't know or care about flex and all the questions and issues I saw were asking about text positioning.

File: Cargo.toml [hunk 1]
```diff
@@ -1726,12 +1726,12 @@ category = "UI (User Interface)"
...
-path = "examples/ui/text_layout.rs"
+name = "flex_layout"
+path = "examples/ui/flex_layout.rs"
 
-[package.metadata.example.text_layout]
-name = "Text Layout"
-description = "Demonstrates how the AlignItems and JustifyContent properties can be composed to layout text"
+[package.metadata.example.flex_layout]
+name = "Flex Layout"
+description = "Demonstrates how the AlignItems and JustifyContent properties can be composed to layout nodes"
```
Comment (alice-i-cecile) [inline Cargo.toml:1734]:
```suggestion
description = "Demonstrates how the AlignItems and JustifyContent properties can be composed to layout nodes and position text"
```

File: examples/README.md [hunk 2]
```diff
@@ -325,11 +325,11 @@ Example | Description
 Example | Description
 --- | ---
 [Button](../examples/ui/button.rs) | Illustrates creating and updating a button
+[Flex Layout](../examples/ui/flex_layout.rs) | Demonstrates how the AlignItems and JustifyContent properties can be composed to layout nodes
```
Comment (alice-i-cecile) [inline examples/README.md:328]:
```suggestion
[Flex Layout](../examples/ui/flex_layout.rs) | Demonstrates how the AlignItems and JustifyContent properties can be composed to layout nodes and position text
```

**Comment by doup** [comment]
I would keep it as `flex_layout` or maybe `nodes_layout`… but that will get old as soon as we add CSS grid. I've merged Alice's suggestion, what do you think of?
> Demonstrates how the AlignItems and JustifyContent properties can be composed to layout nodes **and position text**

**Comment by ickshonpe** [comment]
> I would keep it as `flex_layout` or maybe `nodes_layout`… but that will get old as soon as we add CSS grid. I've merged Alice's suggestion, what do you think of?
> 
> > Demonstrates how the AlignItems and JustifyContent properties can be composed to layout nodes **and position text**

Yep, that looks really good.

**Review by ickshonpe — APPROVED** [review]

**Review by alice-i-cecile — APPROVED** [review]

**Comment by ickshonpe** [comment]
Once grid is implemented, it might be useful to have an equivalent `grid_layout` example that gives identical output but uses grid to create the layout.

**Comment by doup** [comment]
Yup. That was another reason for the rename. ^_^

Also, this example could use a `flex-direction` toggle, to see how changing the main axis direction affects the layout. But that's for another issue/PR.

**Comment by ickshonpe** [comment]
> Yup. That was another reason for the rename. ^_^
> 
> Also, this example could use a `flex-direction` toggle, to see how changing the main axis direction affects the layout. But that's for another issue/PR.

I considered doing this, but I think it might overcomplicate things. It's probably better to have many smaller examples and cover flex-direction by itself.

**Comment by doup** [comment]
@alice-i-cecile this looks like is stuck?

**Pull request merged**